pkg/model/iam/iam_builder.go

```diff
@@ -536,10 +536,12 @@ func addECRPermissions(p *Policy) {
 	})
 }
 
-func addRoute53Permissions(p *Policy, hostedZoneID string) {
+func (b *PolicyBuilder) addRoute53Permissions(p *Policy, hostedZoneID string) {
 
 	// TODO: Route53 currently not supported in China, need to check and fail/return
-
+	//if b.IAMPrefix() == "arn:aws-cn" {
```

Comment:
we can probably remove this commented code right?

Reply:
yeah. I was a little more ambitious when I started but realized I probably shouldn't be adding two things when I only want to fix one bug  :)

Reply:
Ah, I read this as adding a suggestion to the TODO.